Добавляет разметку главной страницы и каталога

VladKolar opened the pull request.

VladKolar committed:
Добавляет разметку главной страницы и каталога

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+            <a class="breadcrumbs-link" href="catalog.html" aria-current="page">Гостиницы</a>
+          </li>
+        </ul>
+      </header>
```
разделение breadcrumbs и фильтров усложнит написание стилей

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+                  Мотель
+                </label>
+              </li>
+              <li class="catalog-filter-form-radiobutton-item">
```
`catalog-filter-form-radiobutton-item` такие подробные именование для классов не нужны

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+            <span class="vissually-hidden">Вид: таблица</span>
+          </button>
+        </li>
+        </ul>
```
невалидная разметка

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,264 @@
+      <a href="https://htmlacademy.ru/" class="main-footer__logo-link">
+        <img src="#" alt="Переход на сайт разработчика" class="main-footer__logo">
+      </a>
+      <footer></footer>
```
`<footer></footer>`

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+      </section>
+      <section class="advantages">
+        <h2 class="visually-hidden">Наши преимущества</h2>
+        <h2 class="sedona-description">Седона - небольшой городок в Аризоне,заслуживаюший большего!</p>
```
нужно оставить один h2, а второй текст обернуть в параграф или просто всего один h2

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+        <p class="index-subscribe-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
+        <form class="index-subscribe-form" action="https://echo.htmlacademy.ru/" method="post">
+          <label class="visually-hidden" for="subscribe-email">E-mail</label>
+          <input class="subscribe-form-email" type="email" placeholder="Ваш e-mail" id="subscribe-email"
```
required

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,145 @@
+      </section>
+      <section class="footer-contacts">
+        <h2 class="visually-hidden">Контакты</h2>
+        <a class="footer-contacts-link" href="Tel:+78128121212" target="_blank" rel="noopener">+7 (812) 812-12-12</a>
```
Tel => tel

VladKolar committed:
Правки по замечаниям + разметка всех страниц

VladKolar committed:
Объединил breadcrumbs и фильтры, изм классы

VladKolar committed:
Добавляет графику для всех страниц

Так-же убрал лишний header в каталоге

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,283 @@
+            <img class="breadcrumbs-item-home" src="images/home.svg" width="12" height="12" alt="На главную">
+            <a class="breadcrumbs-link" href="index.html">Главная</a>
+          </li>
+          <li>
+            <img class="breadcrumbs-item-arrow" src="images/breadcrumbs.svg" width="6.39" height="9.94" alt="Стрелка">
+          </li>
```
эта часть относится к следуюшему пункту и выполняется при помощи css. Можно просто удалить. 
Для width/height допустмы только целые числа

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,283 @@
+        <h2 class="visually-hidden">Список гостиниц</h2>
+        <p class="filtration-result-title">
+          Найдено гостиниц:
+          <span сlass="filtration-result-hotels">38</span>
```
в класс использована кирилица в "с"

Kartaviy-arteom commented on images/Advantage-burger.svg:
именование файлов в соответствии с критериями

Kartaviy-arteom commented on images/Advantage-burger.svg:
именование файлов в соответствии с критериями

Kartaviy-arteom commented on images/star 4.svg:
пробелы в именах недопустимы. Это не одни и теже изображения?

Kartaviy-arteom commented on images/main-subscribe.png:
неверный формат изображения

VladKolar committed:
Исправил названия файлов + базовая стилизация

Изменил названия картинок, убрав заглавные буквы. Добавил базовую стилизацию всех страниц

VladKolar committed:
Delete images/View.svg

Переименовал файл

VladKolar committed:
Удаляет файл images/View-1.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/View-2.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/Advantage-burger.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/Advantage-house.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/Advantage-price.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/Logo-academy.svg

Так как загрузил переименованный файл

VladKolar committed:
Удаляет файл images/star 2.svg

Так как уже есть такой файл

VladKolar committed:
Удаляет файл images/star 3.svg

Так как уже есть такой файл

VladKolar committed:
Удаляет файл images/star 4.svg

Так как уже есть такой файл

VladKolar committed:
Исправил логотип в каталоге, картинки в фонах

Исправил размеры картинок фоновых изображений, поправил цвета текстов в разметке и поправил стилизацию страниц

VladKolar committed:
5.22. Сетка страниц и крупных блоков на флексах

VladKolar committed:
Добавляет сетки страниц на флексах

VladKolar committed:
Сетки компонентов и крупных блоков на флексах

VladKolar committed:
Добавляет сетки компонентов и крупных блоков

VladKolar committed:
Небольшие правки по каталогу, пагинации

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,176 @@
+              812-12-12</a>
+          </section>
+          <section class="footer-logo">
+            <img class="footer-social-vk" src="images/logo-academy.svg" width="115" height="33"
+              alt="Логотип HTML-академии">
```
логотип нужно обернуть ссылкой

VladKolar committed:
Доделал оформление главной страницы и каталога

Сделал главную страницу в соответствии с макетом, сделал страницу каталога, добавил модальное окно, сделал фавиконку, исправил стили для текста, исправил расположение картинок на страницах, добавил паддинги для текстов, сделал раздел с фильтрами и кнопки, исправил ошибки по прошлой проверке

VladKolar committed:
Исправляет название файла
